io.github.lainsce.Quilter.json

```diff
@@ -0,0 +1,106 @@
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "3.38",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.lainsce.Quilter",
+    "rename-icon" : "io.github.lainsce.Quilter",
+    "finish-args" : [
+        "--filesystem=xdg-data/io.github.lainsce.Quilter/",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=network",
+        "--share=ipc"
+    ],
+    "cleanup" : [
+        "/cache",
+        "/man",
+        "/share/aclocal",
+        "/share/devhelp",
+        "/lib/systemd",
+        "/include",
+        "/lib/pkgconfig",
+        "/lib/libvala*",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/vala",
+        "/bin/vapi*",
+        "*.a",
+        "*.la"
+    ],
+    "modules" : [
+        {
+            "name": "gtksourceview",
+            "buildsystem": "meson",
+            "sources": [{
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/gtksourceview/4.7/gtksourceview-4.7.90.tar.xz",
+                "sha256": "983bdcb88f98285b3b997c9335057c8fbc45aad0a7c13cb65eb365567e26baaf"
+            }]
+        },
+        {
+            "name": "gtkspell",
+            "sources": [{
+                "type": "archive",
+                "url": "https://sourceforge.net/projects/gtkspell/files/3.0.10/gtkspell3-3.0.10.tar.xz",
+                "sha256": "b040f63836b347eb344f5542443dc254621805072f7141d49c067ecb5a375732"
+            }],
+            "modules": [{
+                    "name": "enchant",
+                    "sources": [{
+                        "type": "archive",
+                        "url": "https://github.com/AbiWord/enchant/releases/download/v2.2.11/enchant-2.2.11.tar.gz",
+                        "sha256": "a29c5777c4e45fcac2595c15c49d6d2aa434fa5e7c993dff3f9f367b65fe472a"
+                    }]
+                },
+                "shared-modules/intltool/intltool-0.51.json"
+            ]
+        },
+        {
+            "name": "discount",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./configure.sh --prefix=/app --enable-all-features --github-checkbox --with-fenced-code --shared",
+                "sed -e 's|/sbin/ldconfig|/sbin/ldconfig -n|' -i librarian.sh",
+                "make install"
+            ],
+            "sources": [{
+                "type": "archive",
+                "url": "http://www.pell.portland.or.us/~orc/Code/discount/discount-2.2.6.tar.bz2",
+                "sha256": "ae68a4832ff8e620286304ec525c1fe8957be4d8f1e774588eb03d1c3deb74a7"
+            }]
+        },
+        {
+            "name" : "libhandy",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dexamples=false",
+                "-Dprofiling=false",
+                "-Dglade_catalog=disabled",
+                "-Dintrospection=enabled",
```

Comment:
you don't need introspection for vala.

Reply:
Oops, this was me porting the rust flatpak to vala.